sxs/utilities/lvcnr/conversion.py

```diff
@@ -141,64 +157,105 @@ def convert(self, sxs_data_path, out_path, truncation_time=None, resolution=None
         from .waveforms import convert_modes
 
         log = self.Log(self.quiet)
-        log(self.command.format(sxs_data_path=sxs_data_path, out_path=out_path,
-                                truncation_time=truncation_time, resolution=resolution))
-        log("Starting at "+time.strftime('%H:%M%p %Z on %b %d, %Y'))
+        log(
+            self.command.format(
+                sxs_data_path=sxs_data_path, out_path=out_path, truncation_time=truncation_time, resolution=resolution
+            )
+        )
+        log("Starting at " + time.strftime("%H:%M%p %Z on %b %d, %Y"))
 
-        # Load metadata.json from this simulation
-        with open(os.path.join(sxs_data_path, "metadata.json"), 'r') as f:
-            metadata = json.load(f)
+        # Load metadata from this simulation
+        try:
+            metadata = sxs.Metadata.from_file(os.path.join(sxs_data_path, "metadata"))
+        except:
+            raise ValueError("Cannot load metadata.")
 
         # Determine the resolution of the input simulation, if needed
         if resolution is None:
             resolution = sxs.lev_number(sxs_data_path)
         if resolution is None:
-            raise ValueError('No `resolution` value found in input arguments or data path.')
+            raise ValueError("No `resolution` value found in input arguments or data path.")
 
-        sxs_id = sxs_id_from_alt_names(metadata['alternative_names'])
+        sxs_id = sxs_id_from_alt_names(metadata["alternative_names"])
         log("Converting " + sxs_id)
 
         extrapolation_order = "Extrapolated_N2"
         log("Extrapolation order: " + extrapolation_order)
 
-        out_name = out_path + "/" + sxs_id.replace(':', '_') + "_Res" + str(resolution) + ".h5"
+        if not out_waveform_name is None:
+            out_name = out_path + "/" + out_waveform_name
+        else:
+            out_name = out_path + "/" + sxs_id.replace(":", "_") + "_Res" + str(resolution) + ".h5"
         log("Output filename is '{0}'".format(out_name))
 
+        if waveform_name is None:
+            if os.path.exists(os.path.join(sxs_data_path, "Strain_N2.h5")):
+                waveform_path = sxs_data_path + "/Strain_N2"
```

Comment:
```suggestion
                waveform_path = sxs_data_path + "/Strain_N2.h5"
```

Reply:
No `sxs.load` actually seems to fail if you include the .h5 suffix...

Reply:
Hmm.  That sounds like a bug.  Any more detail?

Reply:
It looks like RPDMB just can't find the .json file when the .h5 suffix is included; running `sxs.load('Strain_N2.h5')` for anything in the catalog fails, but `sxs.load('Strain_N2')` works fine. I can post the exact error message if you want.

Reply:
Is this just when you're working with git annex?  If so, I believe I have fixed that issue in my big PR, but for now we could just leave this as a hack.  I would still call it undefined behavior, so it could break in the future, but we could just cross that bridge when we come to it.  Or maybe after my PR merges, we could check again and fix it then.

Reply:
Ah, it'll probably be easier for me to rebase onto this, so I might as well just merge it now.  A functional but missing `.h5` probably won't be the smelliest part of this LVCNR stuff anyway...